src/FormField.js

```diff
@@ -84,31 +84,29 @@ class FormField extends React.Component {
     let iconAdjustment
 
     React.Children.forEach(children, (child, index) => {
-      if (!child) {
-        return
-      }
+      if (!child) return
```

Comment:
A curious question, in anyone has any insight:

If the `React.Children.forEach` top-level API is being used in current code, can one assume that the `child` existence check is handled by the API from React, and this bit of "_safety exit_" code can be omitted?

Reply:
Line 89 will through an error if the child is not an object (React element)